### ISSUE TITLE
question about output_module

### ISSUE THREAD

**Issue by WangliLin** [n-s-f/hierarchical-rnn#10]
In the paper, the output module is a feedforward neural network with two layers, an output embedding layer and a softmax layer.

However, in your code, it is an embedding layer and two feed forward layer which with tanh function as activation function.
